STARTER/js/src/theme.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I don't think we need this demo JS code here. Just a template JS file would be enough